Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,27 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/zipball/master"
+  version "1.34.1"
+  sha256 "775430498a8a38d6bce59e265b948ff47fafb5e00aea3fc5d3c47adcd650a36c"
+
+  depends_on "maven" => :build
+
+  def install
+    libexec.install Dir['*']
+    cd libexec
```

Comment:
You don't need to do this. Just run the `system "mvn", "clean", "install"` before the `libexec.install Dir` stage.